requirements.txt

```diff
@@ -1,6 +1,6 @@
-h5py==2.9.0
+h5py==3.6.0
 nose==1.3.7
 numpy==1.16.3
 PyXB==1.2.6
 six==1.12.0
-xsdata==21.12
\ No newline at end of file
+xsdata==22.2
```

Comment:
We can pin this here. But I don't actually think we are using this file at the moment.

Reply:
I thought this was used for build pipelines?